pkgs/top-level/python-packages.nix

```diff
@@ -6550,6 +6550,8 @@ in {
 
   stevedore = callPackage ../development/python-modules/stevedore {};
 
+  svglib = callPackage ../development/python-modules/svglib {};
```

Comment:
```suggestion
  svglib = callPackage ../development/python-modules/svglib { };
```

Reply:
Whoops, yeah, the package above had the same stylistic issue. Lesson learned!